doc/md/integrations/github-actions.mdx

```diff
@@ -22,16 +22,199 @@ database schema management tasks.
 
 | Action                                                | Use Case                                            |
 |-------------------------------------------------------|-----------------------------------------------------|
-| [ariga/atlas-action](https://github.com/ariga/atlas-action)        | CI for schema changes                               |
-| [ariga/atlas-sync-action](https://github.com/ariga/atlas-sync-action)   | Sync your migration directory to Atlas Cloud (atlasgo.cloud) |
-| [ariga/atlas-deploy-action](https://github.com/ariga/atlas-deploy-action) | Deploy versioned migrations from GitHub Actions    |
 | [ariga/setup-atlas](https://github.com/ariga/setup-atlas) | Install Atlas from a GitHub Actions workflow    |
+| [ariga/atlas-action/migrate/lint](https://github.com/ariga/atlas-action/tree/master/migrate/lint/action.yml)        | CI for schema changes                               |
+| [ariga/atlas-action/migrate/push](https://github.com/ariga/atlas-action/tree/master/migrate/push/action.yml)   | Push your migration directory to Atlas Cloud (atlasgo.cloud) |
+| [ariga/atlas-action/migrate/apply](https://github.com/ariga/atlas-action/tree/master/migrate/apply/action.yml) | Deploy versioned migrations from GitHub Actions    |
 
+## `ariga/setup-atlas`
+
+The [ariga/setup-atlas](https://github.com/ariga/setup-atlas) action can be used to install Atlas from a GitHub Actions workflow.
+
+### Usage
+
+```yaml title=".github/workflows/atlas.yaml"
+name: Deploy schema changes
+run-name: I'm using Atlas 🚀
+on: [push]
+jobs:
+  use-atlas:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: ariga/setup-atlas@v0
+        with:
+          cloud-token: aci_......
+      - run: atlas version
+      # - run: atlas schema apply ...
+```
+
+### Inputs
+
+* `cloud-token` - (Optional) The Atlas Cloud token to use for authentication. To create
+  a cloud token see the [docs](https://atlasgo.io/cloud/bots).
+* `version` - (Optional) The version of the Atlas CLI to install.  Defaults to the latest
+  version.
+
+## `ariga/atlas-action/migrate/push`
+
+Push the current version of your migration directory to Atlas Cloud.
+
+### Usage
+
+<Tabs
+defaultValue="mysql"
+values={[
```

Comment:
I think nowadays sqlite is a valid option as well.

Reply:
<img src="https://media4.giphy.com/media/tHdWFCAd7ssSezUuPq/giphy.gif"/>